Documentation/upgrade.md

```diff
@@ -178,6 +189,19 @@ kubectl -n rook-system get pod ${OPERATOR_POD_NAME} -o jsonpath='{.status.phase}
 Once you've verified the operator is `Running` and on the new version, verify the health of the cluster is still OK.
 Instructions for verifying cluster health can be found in the [health verification section](#health-verification).
 
+##### Allow multiple Mons on one node
+This is needed when you have less nodes than your `mon.count` in  your Cluster CRD.
+Edit the Cluster CRD `kubectl edit -n rook cluster rook`, go to the `mon` spec and set `allowMultiplePerNode` to `true`.
+Please note the default of `allowMultiplePerNode` is `false`.
+```
+spec:
+[...]
+  mon:
+    allowMultiplePerNode: true
```

Comment:
also here should we set the default automatically?

Reply:
The default would be `false`, though I personally think it would break some clusters possibly for users with less than three nodes.
What about mentioning it in the upgrade doc that the user must set `allowMultiplePerNode` to `true` after the operator upgrade if they have less than three nodes?

Reply:
if the upgrade migration just sets it to true, we don't need to document it here, right?